packages/store/src/Slice.sol

```diff
@@ -36,16 +36,24 @@ library SliceLib {
     return Slice.wrap((_pointer << 128) | (data.length & MASK_LEN));
   }
 
+  /**
+   * @dev Subslice a bytes array using the given start index until the end of the array (without copying data)
+   */
+  function getSubslice(bytes memory data, uint256 start) internal pure returns (Slice) {
+    return getSubslice(data, start, data.length);
+  }
```

Comment:
naming nit: what about just `slice`?

Reply:
I named it that way pretty much just to avoid confusion with the `Slice` type, and make searching for it easier